src/Uno.UI.Runtime.Skia.Win32/Win32NativeOpenGLWrapper.cs

```diff
@@ -0,0 +1,155 @@
+using System;
+using System.Runtime.InteropServices;
+
+using Windows.Win32;
+using Windows.Win32.Foundation;
+using Windows.Win32.Graphics.Gdi;
+using Windows.Win32.Graphics.OpenGL;
+using Microsoft.UI.Xaml;
+using Uno.Disposables;
+using Uno.Foundation.Logging;
+using Uno.Graphics;
+using Uno.UI.NativeElementHosting;
+
+namespace Uno.UI.Runtime.Skia.Win32;
+
+// Mostly a copy from WpfNativeOpenGLWrapper
+
+internal class Win32NativeOpenGLWrapper : INativeOpenGLWrapper
```

Comment:
I don't think we have a manual sample in the samples app for this, let's add one.

Reply:
We do, the `GLCanvasElement_***` samples